<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点击复制</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 250, 208);
background-color: rgb(252, 210, 255);</string>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
    <property name="gridSize">
     <size>
      <width>27</width>
      <height>23</height>
     </size>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
    <property name="batchSize">
     <number>100</number>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>反馈属实，已修改完毕。</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>反馈属实，我改一下</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>我改一下，根据</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经核实，词条无误不必修改，感谢反馈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经核实，词条无误，根据</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经检索未找到可靠参考资料，无法修改，请后续提供参考资料佐证</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>问题描述不清晰，无法处理，请后续发布时具体指出。</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>删除词条/版本请前往百科投诉中心提交申请哦http://help.baidu.com/newadd?prod_id=10&amp;category=8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>明星词条人物关系问题请到明星团贴吧提交申请哦https://tieba.baidu.com/p/6087241656 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>同义词解除/合并等，请移步百度百科吧处理专帖下登记哦~https://tieba.baidu.com/p/7902099030</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>锁定词条内容修改请在百科吧专帖下登记申请哦https://tieba.baidu.com/p/6218513321</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ta说文章为自媒体作者撰写，非词条问题科友无法编辑，请点击词条页下方&quot;内容质疑——ta说质疑”提交哦❤️</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>视频为自媒体作者发布，非词条问题科友无法编辑，请点击词条页下方&quot;内容质疑——秒懂质疑”提交哦❤️</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>科星球不处理图片美化更新问题，建议根据百科编辑规则自行处理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>当前概述图无误，无需修改</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>科星球不处理商企类词条问题，建议根据百科编辑规则自行处理哈~~~</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1101</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
